### PR TITLE
update urls for CSI snapshot design doc

### PR DESCRIPTION
**Pull request by boddumanohar**
The original link is broken. Updating with the new CSI Snapshot design doc URLs.

Thanks,
Manohar.

### CODE REVIEW

File: book/src/external-snapshotter.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To use the snapshot beta feature, a snapshot controller is also required. For mo
 
 In the Beta version, the snapshot controller will be watching the Kubernetes API server for `VolumeSnapshot` and `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar only watches the Kubernetes API server for `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar is also responsible for calling the CSI RPCs CreateSnapshot, DeleteSnapshot, and ListSnapshots.
 
-For detailed snapshot beta design changes, see the design doc [here](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190709-csi-snapshot.md).
+For detailed snapshot beta design changes, see the design doc [here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-snapshot.md).
```
Comment (xing-yang) [inline book/src/external-snapshotter.md:25]: Actually it is moved to here: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/177-volume-snapshot/README.md

**Comment by xing-yang** [comment]
Please sign CLA.

**Comment by boddumanohar** [comment]
Sorry for the delay. Updated now.

File: book/src/snapshot-controller.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ For more information on the beta version of the CSI external-snapshotter sidecar
 
 In the Beta version, the snapshot controller will be watching the Kubernetes API server for `VolumeSnapshot` and `VolumeSnapshotContent` CRD objects. The CSI `external-snapshotter` sidecar only watches the Kubernetes API server for `VolumeSnapshotContent` CRD objects. The snapshot controller will be creating the `VolumeSnapshotContent` CRD object which triggers the CSI `external-snapshotter` sidecar to create a snapshot on the storage system.
 
-For detailed snapshot beta design changes, see the design doc [here](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190709-csi-snapshot.md).
+For detailed snapshot beta design changes, see the design doc [here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-snapshot.md).
```
Comment (xing-yang) [inline book/src/snapshot-controller.md:23]: Please update this link as well:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/177-volume-snapshot/README.md

**Comment by boddumanohar** [comment]
Sorry for the delay. Updated now.

**Pull request merged**